src/incubator/TextField/index.tsx

```diff
@@ -46,6 +46,7 @@ const TextField = (props: InternalTextFieldProps) => {
   const {
     modifiers,
     // General
+    containerProps = {},
```

Comment:
because i'm spreading it and IMO this is better:
```
<View {...containerProps} ...>
```
than:
```
<View {...(containerProps ?? {})} ...>
```

WDYT?

Reply:
We can just keep it empty
```
const {
  ...
  contatinerProps,
  ...
  }  = ...

<View {...containerProps} ...>
```